### ISSUE TITLE
Page Unresponsive message in chrome while recording testcase when clicking menu link

### ISSUE THREAD

**Issue by mmraghav** [SeleniumHQ/selenium-ide#1273]
## 🐛 Bug Report

Seeing Page Unresponsive message in chrome browser when we click on a menu during test case recording. If we write the commands in IDE manually instead of recording, Menu click works perfectly fine.

Clicking on menu works perfectly fine when we open the browser and try to click outside of IDE.

## Environment

OS: MacOS
Selenium IDE Version:  3.17.0

Browser:
Browser Version:  chrome 94.0.4606.61

![image](https://user-images.githubusercontent.com/90785815/134703920-a36cbe95-1eda-47bd-8f62-b49b4c1c0bc5.png)

**Comment by toddtarsi** [comment]
@mmraghav - Could you provide more detail here? I am unsure what you mean exactly by "Menu". Can you help provide a screenshot or two or something? Is this still occurring?

**Comment by aliwasif177** [comment]
yes it is still hapening. For normal web it's working fine. BUT , When I run recoring on dom extensive web it makes the page unresponsive.
it shows lots of voilation warnings like mouse hover took 289ms  and souce map not found 404.

![image](https://user-images.githubusercontent.com/60692401/145904742-f4eac72f-c06a-470e-9549-ac8d4dfca7ca.png)


meanwhile  katalon  is tackling it very preciesly using xpath. But in selenium we are using css or id . But in my case this unresponsiveness occurs when we have and <a <script id ='323' >.  </script  > <a/> tag in dom with scipt in it (with id attribute ) 

<img width="1232" alt="Screen Shot 2021-12-14 at 3 07 50 AM" src="https://user-images.githubusercontent.com/60692401/145901993-67b001b5-02d8-4143-8f2f-0be65713918c.png">

**Comment by toddtarsi** [comment]
@aliwasif177 - Interesting, thank you for the further details.

I believe you can use xpath from selenium-ide, if that will resolve your issue.

For the source map issues, that wouldn't impact performance. It's just that there aren't source maps for those assets.

I can be kind of dim witted, so I just want to repeat this back. If you have a (I'm assuming very big) script with an id on it, and you query that script using css or id, it crashes the browser on Selenium IDE. However, if you use Katalon, it works okay?

**Comment by aliwasif177** [comment]
Yes with katalon its working fine even though katalon is built on selenium.
And yes surely I can get xpath with selenium ide.But for that   sqa has to manually get the xpath for that node!

**Comment by aliwasif177** [comment]
Anyway thanks alot for your reply @toddtarsi

**Comment by Ishika-200** [comment]
I am also facing the same issue but can't find any solution. Have you found a solution yet? @aliwasif177

**Comment by toddtarsi** [comment]
@Ishika-200 @aliwasif177 - Do either of you use v4? Do things seem better there?
